Gems/ScriptCanvas/Code/Include/ScriptCanvas/Utils/ScriptCanvasConstants.h

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+#pragma once
+
+#include <AzCore/Name/Name.h>
+
+namespace ScriptCanvas
+{
+    static const AZ::Name RemoteToolsName = AZ::Name::FromStringLiteral("ScriptCanvasRemoteTools");
+    static constexpr AZ::Crc32 RemoteToolsKey("ScriptCanvasRemoteTools");
+    static const uint16_t RemoteToolsPort = 6787;
```

Comment:
This can be constexpr as well.

Also is there any meaning to this port number or was 6787 chosen randomly?